commit 859ebfe200b305925f31fe30e4fc87ab9b8ca79a
Author: Jason King <jason.king@joyent.com>
Date:   2019-04-09T21:50:18+00:00 (6 months ago)
    
    OS-7723 libdemangle missing dependency on libcustr
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/usr/src/lib/Makefile b/usr/src/lib/Makefile
index 669cffa0d3..189f846829 100644
--- a/usr/src/lib/Makefile
+++ b/usr/src/lib/Makefile
@@ -615,6 +615,7 @@ libcmdutils:	libavl
 libcpc:		libpctx
 libcrypt:	libgen
 libctf:		libdwarf
+libdemangle:	libcustr
 libdevid:	libdevinfo
 libdevinfo:	libsec libgen
 libdhcpagent:	libdhcputil libuuid libdlpi libcontract
